image_transport/include/image_transport/image_transport.hpp

```diff
@@ -70,7 +70,8 @@ Subscriber create_subscription(
   const std::string & base_topic,
   const Subscriber::Callback & callback,
   const std::string & transport,
-  rmw_qos_profile_t custom_qos = rmw_qos_profile_default);
+  rmw_qos_profile_t custom_qos = rmw_qos_profile_default,
+  SubscriptionOptionsWithVoidAllocator options = SubscriptionOptionsWithVoidAllocator());
```

Comment:
you can use `rclcpp::SubscriptionOptions` directly:

```suggestion
  rclcpp::SubscriptionOptions options = rclcpp::SubscriptionOptions());
```

Reply:
Great suggestion, I didn't know about this.